sympy/stats/crv_types.py

```diff
@@ -3883,13 +3883,13 @@ def ShiftedGompertz(name, b, eta):
     .. math::
         f(x) := b e^{-b x} e^{-\eta \exp(-b x)} \left[1 + \eta(1 - e^(-bx)) \right]
 
-    with :math: 'x \in [0, \inf)'.
+    with :math:`x \in [0, \infty)`.
```

Comment:
For info: in this situation there is no need to write `:math:` in front as SymPy will default single backtick to equation. (It is also possible to use $ instead of single backtick.)

(No need to update this.)

Reply:
Here I changed that `inf` to `infty` was that correct I am not sure.

Reply:
Yes. \inf is the infimum function and \infty is the infinity symbol. (I checked it, and, yes, \inf does exist. Regarding my comment below.)